charts/spire/charts/spire-server/templates/configmap.yaml

```diff
@@ -1,3 +1,4 @@
+{{ $release := .Release }}
```

Comment:
Why convert this to a variable.

Reply:
Down below the chunk of code using the value is wrapped with a 'with'. That redefines '.' to be what is being 'with'ed.

In order to get access to the top level release object within the 'with' block, it needs to be made an explicit variable.

Reply:
ah yes, scoping. can we move this down to be on top of the block using it?

Reply:
We could. General convention is to put it at the top so that if anything else needs it, it doesn't need to move again.

That ok to do or still think we should move it?

Reply:
Actually, It resolved itself on merge conflict from main. Namespace was added to the top and that's all we need. so I rebased on that.

Reply:
Nah. that didn't work. Had to go the other way. Lets see if this clears the tests.